contrib/hadoop-store-builder/src/java/voldemort/store/readonly/disk/HadoopStoreWriter.java

```diff
@@ -55,6 +55,28 @@
 
     private static final Logger logger = Logger.getLogger(HadoopStoreWriter.class);
 
+    /**
+     * The max chunk size is based on the fact that we use an integer for the bytes
```

Comment:
For each chunk in a partition, Index files contains the integer offset into the corresponding data file. So it is capped at Integer.MAX_VALUE